pkg/apis/openstack/v1alpha1/types_controlplane.go

```diff
@@ -45,3 +48,16 @@ type CloudControllerManagerConfig struct {
 	// +optional
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
 }
+
+// CSI contains configuration for CSI drivers
+type CSI struct {
```

Comment:
Maybe rename to `Storage` [ref](https://github.com/gardener/gardener-extension-provider-aws/blob/b94917b0149e8bb2b2bfe16881a60a4a0a12f03f/pkg/apis/aws/v1alpha1/types_controlplane.go#L50)